statsmodels/genmod/generalized_linear_model.py

```diff
@@ -980,6 +980,11 @@ def fit(self, start_params=None, maxiter=100, method='IRLS', tol=1e-8,
             :math:`rtol * prior + atol > abs(current - prior)`
         tol_criterion : str, optional
             Defaults to ``'deviance'``. Can optionally be ``'params'``.
+        wls_method : str, optional
+            options are 'lstsq', 'pinv' and 'qr'
+            specifies which linear algebra function to use for the irls
+            optimization. Default is `lstsq` which uses the same underlying
+            svd based approach as 'pinv', but is faster during iterations.
```

Comment:
For end users, could you explain why one would use one of the others.

Reply:
I added two sentences but no full explanation.
(AFAIK, we don't have any notes anywhere on the choice of linalg functions)